modules/Core/pages/panel/minecraft_servers.php

```diff
@@ -393,10 +391,9 @@
             $available_parent_servers = DB::getInstance()->get('mc_servers', ['parent_server', 0])->results();
 
             // Display query information alert only if external query is selected
-            $external_query = DB::getInstance()->get('settings', ['name', 'external_query'])->results();
-            $external_query = $external_query[0]->value;
+            $external_query = Util::getSetting('external_query');
```

Comment:
Same opinion here about the setting name (and `$external_query` variable name). These should be something like `$query_type`